android/modules/filesystem/src/java/ti/modules/titanium/filesystem/FilesystemModule.java

```diff
@@ -81,6 +81,10 @@ public boolean isExternalStoragePresent()
 	@Kroll.method
 	public FileProxy getFile(KrollInvocation invocation, Object[] parts)
 	{
+		//If directory doesn't exist, return
+		if (parts[0] == null) {
```

Comment:
Also, I'd say we should log a warning or error to the user to let them know they passed in a null argument (maybe spit out the args received?) and we're returning null. Or throw a JS Error or something so they can pinpoint the bad call and know why they get back a null reference (or get the error thrown).